### PR TITLE
Finished auth module tests

### PR DESCRIPTION
**Pull request by Trinovantes**
This contains tests for
- `loginView`
- `signupView`
- `AuthFormBehavior`

### CODE REVIEW

**Comment by Kapin** [comment]
lgtm

**Comment by Ommy** [comment]
+1

File: src-backbone/tests/auth/signupView_test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        });
+        app = new SanaApp();
+        app._loadBehaviors();
+        let getAppInstance = function() {
+            return app;
+        };
+
+        let LoginView = proxyquire('views/auth/signupView', {
+            'templates/auth/signupView' : {},
+            'sanaAppInstance'          : getAppInstance,
```
Comment (ConnorCimowsky) [inline src-backbone/tests/auth/signupView_test.js]: is this supposed to line up with line 23? :stuck_out_tongue:

**Comment by ConnorCimowsky** [comment]
is good

**Pull request merged**